libs/game/animation.ts

```diff
@@ -567,6 +567,10 @@ namespace animation {
                 return true;
             });
         }
+        if (type == AnimationTypes.All) {
```

Comment:
```suggestion
        if (type == AnimationTypes.All || type == AnimationTypes.ImageAnimation) {
```

Feels like it should also probably apply when you pass `ImageAnimation`, as the options are movement vs picture changing?

Reply:
Yes that would be better. Thanks!